docs/user-guide/gantt-chart/README.md

```diff
@@ -28,17 +29,37 @@ The **Gantt chart** in OpenProject displays the work packages in a timeline. You
 
 ## Activate the Gantt chart
 
-A Gantt chart can be activated in any work package table, to display the work packages in a timeline view.
+A Gantt chart view can be activated either for a specific project, or on the project overarching level if you need to view the timeline for multiple projects. 
+
+For a specific project, select the **Gantt charts** module from the project module menu on the left side. 
+
+![activate-gantt](openproject-user-guide-select-gantt-charts.png)
+
+To view multiple projects in a single timeline, select **Gantt charts** from the **Global Modules** menu. 
```

Comment:
Maybe mention that alternatively, one can also use the "Include projects" feature to add other projects to an existing view?